### ISSUE TITLE
Turbine Camp 

### ISSUE THREAD

**Issue by pdhirajkumarprasad** [nod-ai/SHARK-Turbine#828]
## Introduction 

This page provides information to implement an operator in Shark/IREE front-end. 

## Guidelines

- Pick an ONNX Ops from the list below. 
- Study the specification of the operator you picked on [onnx-ops](https://onnx.ai/onnx/operators/)
- Test that per spec OPs work by writing e2eshark tests and adding tests to the suite
- Try variants, different attribute values
- See what is broken and fix it

Tracking op support for OnnxToTorch lowering. The ops are ordered by priority (highest -> lowest) in each section.

**IMPORTANT:** Mark ops you are working on by hovering over it in the list, then clicking the bullseye symbol to its right. This creates an issue for the op and marks them on the list, avoiding duplicate effort.

Contact:
- @vivekkhandelwal1 (Indian Time Zone) 
- @AmosLewis (US Pacific Time Zone)
- @zjgarvey (US Central Time Zone)
- @marbre  (Central EU Time Zone)

Instructions on adding ONNX or Torch OP:
https://github.com/llvm/torch-mlir/blob/main/docs/add_ops.md

Please add e2e operator level test(s) to e2eshark test suite for your newly added ops using instructions at:
https://github.com/nod-ai/SHARK-TestSuite/blob/main/e2eshark/README.md

If you have questions, please check and ask in the discord [LLVM/torch-mlir](https://discord.gg/btswZzPTVW) channel.

[[Tracker] Onnx FE Support #564](https://github.com/nod-ai/SHARK-Turbine/issues/564)

Onnx Ops: 

 - [ ] torch.aten.linalg_cholesky_ex

**Comment by knwng** [comment]
Seems LSTM also has quality issue. But it seems the testcases themselves are buggy(wrong # of inputs & outputs).

**Comment by vivekkhandelwal1** [comment]
> Seems LSTM also has quality issue. But it seems the testcases themselves are buggy(wrong # of inputs & outputs). Besides, RNN only has 1 testcase rn. I'm afraid it can't cover all the corner cases.

Yeah, the issue for LSTM is already opened here https://github.com/nod-ai/SHARK-ModelDev/issues/315.

**Comment by habcode** [comment]
Hello @zjgarvey @pdhirajkumarprasad , I would like to contribute to this repository and thinking of taking ownership of Cast or
 DequantizeLinear OP. Would you please share some step by step guide on how to get started?

**Comment by AmosLewis** [comment]
https://github.com/nod-ai/SHARK-ModelDev/issues/886
